app/views/docs/authentication-management.phtml

```diff
@@ -0,0 +1,192 @@
+<p>
+    Appwrite has built-in features to help manage user accounts.
+    Users can be organized into teams and be given labels, so they can be given different permissions and access different resources.
+    Each user can also have their own preference object, which you can use to save preferences such as theme, language, and notification settings.
+</p>
+
+[TODO @steven: TEAMS docs]
+
+[TODO @steven: Labels docs]
```

Comment:
Labels are a good way to flag a user to grant them access to resources. For example, a `subscriber` label can be added to a user once they've purchased a subscription:

```javascript
const sdk = require('node-appwrite');

const client = new sdk.Client();

const users = new sdk.Users(client);

users.updateLabels(
  '[USER_ID]',
  [
    sdk.Role.label('subscriber'),
  ]
);
```

And a collection can be be given read access to users with that label:

```javascript
const sdk = require('node-appwrite');

const client = new sdk.Client();

const databases = new sdk.Databases(client);

databases.updateCollection(
  '[DATABASE_ID]',
  '[COLLECTION_ID]',
  '[NAME]',
);
```